test/metrics/test_metrics_engine.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import fairlearn.metrics as metrics
+
+# ===========================================================
+
+# Conversions from Python lists to our supported datatypes
+
+
+def identity(X):
```

Comment:
It seems like we defined all of these somewhere already, unless that PR was deleted or changed. Can we maybe put them in a central place?

Reply:
Sorry, "all of these" meaning these functions "identity", "tondarray", etc.

Reply:
They're slightly different this time, since these take lists not ndarrays. Agreed that we need to standardise all this, possibly in conjunction with the interpret library for handling different input types.

Reply:
OK great, in that case ignore for now